### ISSUE TITLE
Suggestion: Connection status on secondary output

### ISSUE THREAD

**Issue by Sungtroll** [bisand/node-red-contrib-tibber-api#46]
It would be nice to have a way of getting the status light status available on a secondary output.

Example:
![Connection Status](https://user-images.githubusercontent.com/37970110/112351746-71797600-8cca-11eb-83d9-5f46d6dec2ef.png)

**Comment by bisand** [comment]
I suggest you instead use the Node-Red status node. This is intended to report status changes for selected nodes in your flow. See the attached screenshot.
![image](https://user-images.githubusercontent.com/376963/112431155-70366080-8d3f-11eb-8790-3dc657982939.png)

**Comment by Sungtroll** [comment]
I got it to work with the status node and the browser now speak out status if status changed and log to text file.

I want to share with you how to get the modern browser based on chromium to stay alive to get alerts trough node-red ui. If not using this settings the tab will be unloaded or go to sleep and stop notifying if tab is not in focus. I have reported it as unwanted behavior and hopefully changes will be made to make browser reliable for notifications even if tab is not in focus.

Here is the temporary fix:
HOW TO: Make the dashboard run at all time in Chrome and Edge beta
![image](https://user-images.githubusercontent.com/37970110/112506999-920b0400-8d8e-11eb-938f-951b0deaccdf.png)
